packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -816,6 +816,89 @@ class AndroidCameraCameraX extends CameraPlatform {
     await cameraControl.setZoomRatio(zoom);
   }
 
+  @override
+  Future<Iterable<VideoStabilizationMode>> getVideoStabilizationSupportedModes(
+      int cameraId) async {
+    final List<int> modeIndexes =
+        await _getAvailableVideoStabilizationModeIndexes();
+
+    final List<VideoStabilizationMode> list = <VideoStabilizationMode>[];
+    for (final int ix in modeIndexes) {
+      final VideoStabilizationMode? mode = switch (ix) {
+        0 => VideoStabilizationMode.off,
+        1 => VideoStabilizationMode.on,
+        2 => VideoStabilizationMode.standard,
+        _ => null,
+      };
+      if (mode != null) {
+        list.add(mode);
+      }
+    }
+    return list;
+  }
+
+  /// Set the video stabilization mode for the selected camera.
+  @override
+  Future<void> setVideoStabilizationMode(
+      int cameraId, VideoStabilizationMode mode) async {
+    final int? controlMode = _getControlVideoStabilizationMode(mode);
```

Comment:
If `_getAvailableVideoStabilizationModeIndexes` returns `VideoStabilizationMode`s like I suggested above, this call may not be needed too.